product_brand/product_brand_view.xml

```diff
@@ -129,18 +129,19 @@
             </field>
         </record>
 
+<!--Useless views since in v9 personnal groups are possible-->
```

Comment:
Not really useless, because they're directly "groupable" instead having to search the field in the field list.